apps/arena/lib/arena/game_updater.ex

```diff
@@ -53,6 +56,8 @@ defmodule Arena.GameUpdater do
   def handle_info(:update_game, %{game_state: game_state} = state) do
     Process.send_after(self(), :update_game, state.game_config.game.tick_rate_ms)
 
+    now = DateTime.utc_now() |> DateTime.to_unix(:millisecond)
```

Comment:
What do you think of using `System.monotonic_time(:millisecond)` instead? 

Does the client need this timestamp?